src/platform/utilities/data/camelCaseObjectKeys.js

```diff
@@ -1,14 +1,14 @@
-import _ from 'lodash';
+import { isPlainObject, camelCase, reduce, isArray } from 'lodash';
```

Comment:
^^ https://github.com/department-of-veterans-affairs/vets-website/pull/11673/files#r377317714